packages/core/http/core-http-server-internal/src/http_config.ts

```diff
@@ -46,7 +46,7 @@ const validHostName = () => {
  * We assume the URL does not contain anything after the pathname so that
  * we can safely append values to the pathname at runtime.
  */
-function validateCdnURL(urlString: string): undefined | string {
+export const validateCdnURL = (urlString: string) => {
```

Comment:
Style, use arrow funcs to match rest of file and dropped return type annotation (rather use inference).